magento/magento2-login-as-customer
# magento/magento2-login-as-customer#150: Opt-in/out V3.

### Description (*)


### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/304


### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/magento2/issues/29113: Customers should be able to enable/disable Admin "remote shopping assistance". 
2. Fixes https://github.com/magento/magento2/issues/29114: Admin is able to change "Allow remote shopping assistance" option without access to "Change customer opt-in preference"

### Manual testing scenarios (*)

1. See https://github.com/magento/magento2/issues/29113: Customers should be able to enable/disable Admin "remote shopping assistance". 
2. See https://github.com/magento/magento2/issues/29114: Admin is able to change "Allow remote shopping assistance" option without access to "Change customer opt-in preference"

### Questions or comments


### Contribution checklist (*)
